transport/tls-record-frag/writer.go

```diff
@@ -0,0 +1,105 @@
+package tlsrecordfrag
+
+import (
+	"errors"
+	"io"
+)
+
+type tlsRecordFragWriter struct {
+	writer      io.Writer
+	prefixBytes uint32
```

Comment:
Unsigned ints are dangerous.
```suggestion
	prefixBytes int32
```